packages/pn-pa-webapp/src/pages/components/Notifications/FilterNotifications.tsx

```diff
@@ -69,6 +69,33 @@ function isFilterapplied(filtersCount: number): boolean {
 
 const getValidStatus = (status: string) => (status === 'All' ? '' : status);
 
+// the formik validations (which control the enable status of the "filtra" button)
+// must coincide with the input field validations (which control the color of the frame around each field)
+const dateFieldsValidation = {
+  // startDate cannot be earlier than 10 years ago
+  startDate: yup.date().min(tenYearsAgo).test({
```

Comment:
why not yup.date().min(tenYearsAgo).max(today) ?

Reply:
ok, done as suggested